ui/src/app/views/admin-user.spec.tsx

```diff
@@ -0,0 +1,41 @@
+import {mount} from 'enzyme';
+import * as React from 'react';
+
+import {AdminUser} from './admin-user';
+import {AuthDomainApi, Profile, ProfileApi} from 'generated/fetch';
+import {serverConfigStore} from 'app/utils/navigation';
+import {ProfileApiStub, ProfileStubVariables} from 'testing/stubs/profile-api-stub';
+import {registerApiClient} from 'app/services/swagger-fetch-clients';
+import {AuthDomainApiStub} from 'testing/stubs/auth-domain-api-stub';
+
+
+describe('AdminUser', () => {
+  let props: {profile: Profile};
+
+  const component = () => {
+    return mount(<AdminUser {...props}/>);
+  };
+
+  beforeEach(() => {
+    serverConfigStore.next({
```

Comment:
Are all of these config properties actually relevant to the admin-user test setup? If not, it might be really useful to create a helper method somewhere that can be used by multiple tests to create a bog-standard ServerConfig for testing. This will help reduce the # of file edits required if we e.g. remove one of these config flags in the future.